index.js

```diff
@@ -194,6 +195,10 @@ function getDefaultConfig(mainTemplate, templatesPath) {
         return Array.isArray(value);
       };
 
+      ramlObj.stringify = function (value) {
+        return typeof value === 'object' ? yaml.dump(value) : value;
```

Comment:
@tr3ddy that's old way for this PR (https://github.com/raml2html/raml2html/pull/210#issuecomment-222116988), and please see https://github.com/raml2html/raml2html/pull/210#issuecomment-224514408